### DISCUSSION TITLE
Importing Map as Images

### DISCUSSION THREAD

**Discussion by EzSalvucci** [orbat-mapper/orbat-mapper#162] [Q&A] [answered]
Hi, Kjell.

I was wandering, there is a way to import a map as an image (like a .png)?

**Comment by kjellmf** [comment]
Not yet, but I have added an issue for this #149.

**Comment by kjellmf** [comment]
@EzSalvucci Just curious. Do you have georeferenced images that you want to add? How large will they typically be?

**Reply by EzSalvucci** [reply]
Yes, I have a map of Ohau in Hawaii in 1881, and it's big.
![3706000](https://github.com/orbat-mapper/orbat-mapper/assets/119259569/77c85cc8-2a09-46ab-b4b8-c17520fd8e45)

**Comment by kjellmf** [comment, marked as answer]
@EzSalvucci I have now implemented basic support for importing images. It works best if your image is hosted online somewhere, but you can also load a local image.
